examples/metal-2-fabric-aws-connection/terraform.tfvars.example

```diff
@@ -1,5 +1,6 @@
 equinix_client_id      = "MyEquinixClientId"
 equinix_client_secret  = "MyEquinixSecret"
+equinix_endpoint       = "endpoint_URL"
```

Comment:
I don't think we need to add this for the customer. They can use this to switch to the sandbox environment but if they aren't interested in this then they may think that they need to set this when it already has a default.